src/test/resources/spec/concordion/annotation/Resources.html

```diff
@@ -0,0 +1,218 @@
+@Resources( "demo.css" )
+public class ResourcesDemoTest {
+    
+}
+		</pre>
+		
+		<p concordion:execute="#result1 = process(#javaFragment)">will include a link to the resource <span concordion:assertEquals="getLink(#TEXT)">demo.css</span> into the generated specification
+		and leave the <span concordion:assertTrue="isDefaultCssIncluded()">default CSS</span> in the generated specification.</p>
+	</div>
+
+	<!-- ============================================= -->
+    <div class="example">
+        <h3>Example: Single Embedded Resource</h3>
+		
+		<p>Executing the following fixture:</p>
+		
+		<pre class="java" concordion:set="#javaFragment">
+import org.concordion.api.Resources;
+import org.concordion.api.Resources.InsertType;
+import org.concordion.integration.junit4.ConcordionRunner;
+import org.junit.runner.RunWith;
+
+@RunWith(ConcordionRunner.class)
+@Resources( value = "demo.css", insertType = InsertType.EMBEDDED )
+public class ResourcesDemoTest {
+    
+}
+		</pre>
+	
+    	<p concordion:execute="#result1 = process(#javaFragment)">will embed the resource <span concordion:assertTrue="isCssIncluded(#TEXT)">demo.css</span> into the generated specification.</p>
+    </div> 
+
+	<!-- ============================================= -->
+	<div class="example">
+        <h3>Example: Remove Default CSS</h3>
+		
+		<p>Executing the following fixture:</p>
+		
+		<pre class="java" concordion:set="#javaFragment">
+import org.concordion.api.Resources;
+import org.concordion.api.Resources.InsertType;
+import org.concordion.integration.junit4.ConcordionRunner;
+import org.junit.runner.RunWith;
+
+@RunWith(ConcordionRunner.class)
+@Resources( value = "demo.css", includeDefaultStyling = false )
+public class ResourcesDemoTest {
+    
+}
+		</pre>
+	
+    	<p concordion:execute="#result1 = process(#javaFragment)">will include a link to the resource <span concordion:assertEquals="getLink(#TEXT)">demo.css</span> 
+    		and remove the <span concordion:assertFalse="isDefaultCssIncluded()">default CSS</span> from the generated specification.
+    	</p>
+    </div> 
+    
+	<!-- ============================================= -->
+	<div class="example">
+        <h3>Example: Multiple Resources, Types and Locations</h3>
+		
+		<p>Executing the following fixture:</p>
+		
+		<pre class="java" concordion:set="#javaFragment">
+import org.concordion.api.Resources;
+import org.concordion.api.Resources.InsertType;
+import org.concordion.integration.junit4.ConcordionRunner;
+import org.junit.runner.RunWith;
+
+@RunWith(ConcordionRunner.class)
+@Resources( value = { "demo.css", "/spec/demo.js", "/spec/demo.txt" }, includeDefaultStyling = false )
+public class ResourcesDemoTest {
+    
+}
+		</pre>
+	
+    	<p concordion:execute="#result1 = process(#javaFragment)">will include 
+    		a link to the resource <span concordion:assertEquals="getLink(#TEXT)">demo.css</span>,
+    		a script link for <span concordion:assertEquals="getScript(#TEXT)">demo.js</span>,
```

Comment:
This was to show linking to a resource from the root folder as opposed to from the class folder. I've added an example to highlight this